cni/network/network_windows_test.go

```diff
@@ -19,6 +20,10 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func init() {
+	network.Hnsv2 = hnswrapper.NewHnsv2wrapperFake()
+	network.Hnsv1 = hnswrapper.Hnsv1wrapperfake{}
```

Comment:
lets keep it consistent..either have New function like hnsv2 or remove function and intialize hnsv2 directly